src/ameba.ts

```diff
@@ -124,11 +184,16 @@ export class Ameba {
         }
     }
 
-    public clear(document: TextDocument): void {
-        let uri = document.uri;
-        if (uri.scheme === 'file') {
-            this.taskQueue.cancel(uri);
-            this.diag.delete(uri);
+    public clear(document: TextDocument | null = null): void {
```

Comment:
how the document can be null here?

Reply:
Passing null is used for clearing all diagnostics